### PR TITLE
Add core transaction:send command - Closes #272

### LINKED ISSUE #272: Add core transaction:send command
### Description

Add `transaction:send` command, accepting  `transaction` as a parameter in base64 format, to send a specified transaction to the local node, according to the following specification:

1. It should produce an error if the node specified in the data path is not running
2. It should support:

```
--data-path (-d / LISK_DATA_PATH)
  Specifies which data path the application should use
```

### Motivation

Allow the broadcasting of transactions via the local node.

### Acceptance Criteria

- Add unit tests
- Examples and description are clear and correct
- Check if the transaction is accepted by the application

### Additional Information

Related issues: #254

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

This PR resolves #272

### How was it solved?

By adding the command to send a transaction to the node by IPC

### How was it tested?

- Unit tests
- Create a transaction and get the encoded base64 representation then run the command. Example: 
```
./bin/run transaction:send CAgQABiAreIEIiAP6aPxohtVMPJ/h6QUtUnnmpQL8k/fKy8F5/Iq7uzIaioeCIDIr6AlEhTKU+nitO+qcqESWBGjhj99dxWx/hoAMkBI5kK8+Hn3/XeFXavYARJWL0SI4Pc0Pky5CJrX8rKy6FgLzwQoG1dyDrw7DszWhG+vpF6LZ/XOC/H772gDKJ0G --data-path=/Users/pablo/.lisk/devnet/
```

you should get a message `Transaction with id: 'cpKrfcuFVfkjw2QK3hLmanu4v73flpI58Doa0WxK3us=' received by node`

This PR depends on https://github.com/LiskHQ/lisk-sdk/pull/5597 so switch and build its branch or wait for it to be merged before testing.

### CODE REVIEW

File: src/commands/transaction/send.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	};
+
+	static args = [
+		{ name: 'transaction', required: true, description: 'The transaction to be sent to the node' },
+	];
+
+	async run(): Promise<void> {
+		const {
+			args: { transaction },
+		} = this.parse(this.constructor as typeof SendCommand);
```
Comment (ManuGowda) [inline src/commands/transaction/send.ts:30]:
```suggestion
		} = this.parse(SendCommand);
```

File: src/commands/transaction/send.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		if (!validator.isBase64String(transaction)) {
+			throw new Error('The transaction must be provided as a base64 encoded string');
+		}
+
+		const { transactionId } = await this._channel.invoke('app:postTransaction', { transaction });
+		this.log(`Transaction with id: '${transactionId as string}' received by node`);
```
Comment (ManuGowda) [inline src/commands/transaction/send.ts:36]: no try catch?

File: test/commands/transaction/send.spec.ts [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+import { expect, test } from '@oclif/test';
+import * as sandbox from 'sinon';
+import * as fs from 'fs-extra';
+import { join } from 'path';
+import { IPCChannel } from 'lisk-sdk';
+
+import * as appUtils from '../../../src/utils/application';
+import { createTransferTransaction, encodeTransactionFromJSON } from '../../utils/transactions';
+
+export const baseTransactionSchema = {
```
Comment (ManuGowda) [inline test/commands/transaction/send.spec.ts:25]: no need to export?

File: src/commands/transaction/send.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import { validator } from 'lisk-sdk';
+import BaseIPCCommand from '../../base_ipc';
+
+export default class SendCommand extends BaseIPCCommand {
+	static flags = {
+		...BaseIPCCommand.flags,
+	};
+
+	static args = [
+		{ name: 'transaction', required: true, description: 'The transaction to be sent to the node' },
```
Comment (shuse2) [inline src/commands/transaction/send.ts:24]:
```suggestion
		{ name: 'transaction', required: true, description: 'The transaction to be sent to the node in base64 string' },
```

File: src/commands/transaction/send.ts (hunk 1 above)
Comment (shuse2) [inline src/commands/transaction/send.ts:30]:
```suggestion
		} = this.parse(SendCommand);
```

File: src/commands/transaction/send.ts (hunk 2 above)
Comment (pablitovicente) [reply src/commands/transaction/send.ts:36]: It's not necessary as it seems oclif catches it and in fact I suggested to remove it on a different PR where I had added the try/catch and then removed based on thee suggestion.

File: test/commands/transaction/send.spec.ts (hunk 3 above)
Comment (pablitovicente) [reply test/commands/transaction/send.spec.ts:25]: Should I remove from other tests? Thought we were exporting them for some future usage.

File: src/commands/transaction/send.ts (hunk 1 above)
Comment (pablitovicente) [reply src/commands/transaction/send.ts]: Committed as part of other suggestions.

**Pull request merged**